promscale/quick-start.md

```diff
@@ -0,0 +1,97 @@
+# Promscale quick start
+You can use Docker Compose to easily run Promscale in an isolated environment built
+with Docker containers. This quick start guide demonstrates how to use Docker Compose to set
+up and run Promscale as a single data store for your metrics and traces with full Prometheus
+and OpenTelemetry support. It also includes some additional pre-configured tooling for you
+to get familiar with additional Promscale features, such as performance monitoring and
+visualization tools.
+
+To learn more about how Promscale can improve your observability stack, see
+[About Promscale][about-promscale]. For instructions on installing Promscale directly, see
+[Installing Promscale][install-promscale].
+
+Before you begin, make sure that you have [installed Docker Compose][docker-compose].
+
+## Install Promscale with Docker Compose
+To install Promscale with Docker Compose, you need to start by cloning the Promscale 
+[repository][gh-promscale], and then using the Docker Compose tool to install it. The 
+`docker-compose` directory contains services for these components:
+* Promscale for analytics and long term storage of metrics and traces
+* Prometheus and Node exporter for metrics
+* OpenTelemetry collector, demonstration applications, Jaeger query and Grafana for traces and visualization
+
+### Installing Promscale with Docker Compose
+
+<procedure>
+
+1. Clone the repository that defines the services for Promscale:
+   ```bash
+   git clone git@github.com:timescale/promscale.git
+   ```
+1. Change to the `promscale-demo` directory:
+   ```bash
+   cd promscale/docker-compose/promscale-demo
+   ```
+1. Use Docker Compose to run Promscale:
+   ```bash
+   docker compose up -d
+   ```  
+
+</procedure >
+
+## Explore data in Promscale
```

Comment:
We have to add Jaeger here. At the very least just provide the url. Also make the Jaeger url and the Grafana url links, so folks can just click instead of having to copy and paste.